pkg/tfvars/azure/azure.go

```diff
@@ -27,27 +26,49 @@ type config struct {
 	ImageURL                    string            `json:"azure_image_url,omitempty"`
 	Region                      string            `json:"azure_region,omitempty"`
 	BaseDomainResourceGroupName string            `json:"azure_base_domain_resource_group_name,omitempty"`
+	NetworkResourceGroupName    string            `json:"azure_network_resource_group_name,omitempty"`
```

Comment:
since these are required, should not have omitempty.

also the all of the below ones too.

Reply:
Fixed!